src/resources/Enums.ts

```diff
@@ -706,3 +706,11 @@ export enum FormInputType {
     Password = 'Password',
     Input = 'Input',
 }
+
+export enum AssociationModelTypes {
```

Comment:
[optional] Is it better to name it simply `ModelTypes`, which is more generic, since we could use this enum in both SO and ML.